src/Ryujinx.Ava/UI/Windows/CheatWindow.axaml.cs

```diff
@@ -119,5 +124,33 @@ public void Save()
 
             Close();
         }
+
+        public void OpenCheatFolder()
+        {
+            string modsBasePath = ModLoader.GetModsBasePath();
+            string cheatsPath = Path.Combine(modsBasePath, "contents", TitleId, "Cheats");
```

Comment:
```suggestion
            string cheatsPath = Path.Combine(modsBasePath, "contents", TitleId, "cheats");
```

Why not restore the previous buttons? They were better imo.